src/jit/codegenlinear.h

```diff
@@ -219,6 +223,15 @@ bool genIsRegCandidateLocal(GenTreePtr tree)
     return (varDsc->lvIsRegCandidate());
 }
 
+#ifdef FEATURE_PUT_STRUCT_ARG_STK
+#ifdef _TARGET_X86_
+bool m_pushStkArg;
+#else // !_TARGET_X86_
+unsigned m_stkArgVarNum;
+#endif // !_TARGET_X86_
+unsigned m_stkArgOffset;
+#endif // !FEATURE_PUT_STRUCT_ARG_STK
```

Comment:
I went down that path at first, but I felt that it was too pervasive. This seems to localize the issues.

---
In reply to: [79526928](https://github.com/dotnet/coreclr/pull/7252#discussion_r79526928) [](ancestors = 79526928)